data-catalog-backend-test/data-catalog-backend-test-component/src/test/java/no/nav/data/catalog/backend/test/component/informationtype/InformationTypeControllerTest.java

```diff
@@ -63,17 +69,42 @@ public void setup() {
 		informationType = InformationType.builder()
 				.id(1L)
 				.name("Test")
-				.description("Test av brukerinput")
+				.description("Test description")
 				.category("PERSONALIA")
-				.producer("BRUKER")
+				.producer("SKATTEETATEN")
 				.system("TPS")
 				.personalData(true)
 				.elasticsearchId(base64UUID())
 				.elasticsearchStatus(ElasticsearchStatus.TO_BE_CREATED)
 				.build();
+		informationType.setCreatedBy("Mr Melk");
+		informationType.setCreatedDate(new Date());
+
+//		informationTypeResponse = informationType.convertToResponse();
```

Comment:
Remove the commented code.

Reply:
Relatert til tester som er ignorert. Må se mer på hvordan jeg kan få disse to tester å kjøre